network/router.go

```diff
@@ -22,7 +22,7 @@ import (
 //   router.Stop() // will stop the listening and the managing of all Conn
 type Router struct {
 	// id is our own ServerIdentity
-	id *ServerIdentity
+	ServerIdentity *ServerIdentity
```

Comment:
why do you export it ? Is it necessary ? I don't think it's meant to change...
On another matter, we should rethink our rule "use pointers always" because I had some problems with ServerIdentity pointers, when you change the address (for listening on port 0 for example). One day or another, it will bite us...